### PR TITLE
WIP: LOOC

### PR DESCRIPTION
**Pull request by Kmsxkuse**
:cl: Kmsxkuse
Adds: LOOC, limited range ooc.
/:cl:

I think it works. Testing now.

### CODE REVIEW

File: code/modules/client/verbs/ooc.dm [hunk 1]
```diff
@@ -259,3 +259,73 @@ GLOBAL_VAR_INIT(normal_ooc_colour, OOC_COLOR)
 		to_chat(src, "You can't ignore yourself.")
 		return
 	ignore_key(selection)
+
+/client/verb/looc(msg as text)
```
Comment (ninjanomnom) [inline code/modules/client/verbs/ooc.dm]: Why is this verb duplicated

File: code/__DEFINES/admin.dm [hunk 2]
```diff
@@ -3,10 +3,11 @@
...
 #define MUTE_OOC		2
-#define MUTE_PRAY		4
-#define MUTE_ADMINHELP	8
-#define MUTE_DEADCHAT	16
-#define MUTE_ALL		31
+#define MUTE_LOOC   4
+#define MUTE_PRAY		8
+#define MUTE_ADMINHELP	16
+#define MUTE_DEADCHAT	32
+#define MUTE_ALL		64
```
Comment (ninjanomnom) [inline code/__DEFINES/admin.dm]: MUTE_ALL needs to be 63, it's a bit value

Comment (ninjanomnom) [inline code/__DEFINES/admin.dm]: `1000000` is 64
`111111` is 63

**Comment by Qustinnus** [comment]
Why do we need this?
1. it reinforces the crappy "LOOC: HAHAHA omg haha so funny" and "LOOC: OMG FUCK U I RPEORT TU ADMIN!!" behavior

2. If you have a question you can Ahelp or soon-ish ask a mentor.

I just don't see enough reasons for us to have LOOC.

**Comment by Kmsxkuse** [comment]
Alright, shelving this for some later time. It's hard enough already to get it working anyways.

**Pull request closed without merging**